src/features/Respondents/Respondents.tsx

```diff
@@ -104,12 +109,43 @@ export const Respondents = () => {
         filterRows(secretId, searchValue) || filterRows(nickname, searchValue),
     );
 
+  const chosenRespondentsItems =
+    respondentsDataIndex || respondentsDataIndex === 0
+      ? usersData?.items[respondentsDataIndex]
+      : undefined;
+
   const appletsSmallTableRows = getAppletsSmallTableRows(
-    usersData?.items[respondentsDataIndex],
+    chosenRespondentsItems,
     appletsData,
     setChosenAppletData,
   );
 
+  useEffect(() => {
+    if (chosenRespondentsItems && Object.keys(chosenRespondentsItems).length === 1) {
```

Comment:
you can use  it once Object.keys(chosenRespondentsItems)

Reply:
const keys = chosenRespondentsItems && Object.keys(chosenRespondentsItems), this way ?

Reply:
I think it's less more equal to what I have now.